python/tvm/topi/nn/sparse.py

```diff
@@ -470,6 +470,38 @@ def _traverse(t):
     return sparse_input_map
 
 
+def random_bsr_matrix(m, n, bs_r, bs_c, density, dtype):
```

Comment:
I O this should not be part of Topi. Either you can put where it is used or I testing.

Reply:
> IMO this should not be part of Topi. Either you can put where it is used or I testing.

Fine, just I'm finding that this has been used in many different places. I'll try to find another postion.

Reply:
Moved to `topi/sparse/utils`.

Reply:
I am sorry for such delayed response, i missed your reply somehow. What my suggestion is, random_bsr_matrix() does not qualify to be in Topi unless it is required by some Ops. What i could see it is just utility for Tutorial, so lets keep this utility func in Tutorial file itself. Otherwise we have one more option, we can put it as part of tvm.testing which can help other tutorials and testcases as well.